src/Data/Formatter/DateTime.purs

```diff
@@ -114,6 +111,18 @@ parseFormatString ∷ String → Either String Formatter
 parseFormatString s =
   lmap P.parseErrorMessage $ P.runParser s formatParser
 
+-- | Formatting function that accepts a number that is a year,
+-- | and strips away the non-significat digits, leaving only the
+-- | ones and tens positions.
+formatYearTwoDigits :: Int -> String
+formatYearTwoDigits i = case dateLength of
+  1 -> "0" <> (show $ abs i)
+  2 -> show $ abs i
+  _ -> reverse $ Str.take 2 $ reverse $ show $ abs i
+  where
+    dateLength = Str.length $ show $ abs i
+    reverse str = foldr (flip (<>)) "" (Str.split (Str.Pattern "") str)

```

Comment:
Almost there I think! I'd suggest storing `dateString = show (abs i)` or something like that too, since it's reused a bunch in here.

For the `_` case it could just be `Str.drop (Str.length dateString - 2) dateString` too then.